### PR TITLE
New CTDMO cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PY VI Instrument Cal CSVs [DT-219]](https://trello.com/c/CkLoKMMl/219-py-vi-instrument-cal-csvs-dt-219)

File: calibration/CTDMOG/CGINS-CTDMOG-12372__20240128.csv
Comment (sauuyer) [inline calibration/CTDMOG/CGINS-CTDMOG-12372__20240128.csv:1]: Missing black space between "source_file" and file name

**Review by sauuyer — COMMENTED** [review]
Otherwise, this one looks good

File: calibration/CTDMOH/CGINS-CTDMOH-11704__20240126.csv
Comment (sauuyer) [inline calibration/CTDMOH/CGINS-CTDMOH-11704__20240126.csv:1]: Looks good.

File: calibration/CTDMOH/CGINS-CTDMOH-12225__20240128.csv
Comment (sauuyer) [inline calibration/CTDMOH/CGINS-CTDMOH-12225__20240128.csv:1]: Looks good.

File: calibration/CTDMOH/CGINS-CTDMOH-12386__20240131.csv
Comment (sauuyer) [inline calibration/CTDMOH/CGINS-CTDMOH-12386__20240131.csv:1]: Looks good.

File: calibration/CTDMOG/CGINS-CTDMOG-12379__20240130.csv [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+37-12379,CC_ptca0,5.245150e+005,
+37-12379,CC_ptca1,2.328367e-001,
+37-12379,CC_ptca2,3.554811e-003,
+37-12379,CC_ptcb0,2.535438e+001,
+37-12379,CC_ptcb1,8.750000e-004,
+37-12379,CC_ptcb2,0.000000e+000,
+37-12379,CC_ptempa0,-6.843094e+001,
+37-12379,CC_ptempa1,5.178428e-002,
+37-12379,CC_ptempa2,-5.697338e-007,
+37-12379,CC_p_range,1450,
```
Comment (reedan88) [inline calibration/CTDMOG/CGINS-CTDMOG-12379__20240130.csv:25]: Looks good.

File: calibration/CTDMOG/CGINS-CTDMOG-12396__20240126.csv [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+37-12396,CC_ptca0,5.242477e+005,
+37-12396,CC_ptca1,3.586833e+000,
+37-12396,CC_ptca2,-2.245864e-002,
+37-12396,CC_ptcb0,2.530663e+001,
+37-12396,CC_ptcb1,-6.750000e-004,
+37-12396,CC_ptcb2,0.000000e+000,
+37-12396,CC_ptempa0,-6.547413e+001,
+37-12396,CC_ptempa1,5.092204e-002,
+37-12396,CC_ptempa2,-2.567684e-007,
+37-12396,CC_p_range,1450,
```
Comment (reedan88) [inline calibration/CTDMOG/CGINS-CTDMOG-12396__20240126.csv:25]: Looks good.

File: calibration/CTDMOG/CGINS-CTDMOG-12400__20240130.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+37-12400,CC_ptca0,5.248258e+005,
+37-12400,CC_ptca1,1.849123e+000,
+37-12400,CC_ptca2,1.346690e-001,
+37-12400,CC_ptcb0,2.534375e+001,
+37-12400,CC_ptcb1,-1.250000e-003,
+37-12400,CC_ptcb2,0.000000e+000,
+37-12400,CC_ptempa0,-6.823257e+001,
+37-12400,CC_ptempa1,5.099421e-002,
+37-12400,CC_ptempa2,-4.378940e-007,
+37-12400,CC_p_range,1450,
```
Comment (reedan88) [inline calibration/CTDMOG/CGINS-CTDMOG-12400__20240130.csv:25]: Looks good.

File: calibration/CTDMOG/CGINS-CTDMOG-12403__20240124.csv [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+37-12403,CC_ptca0,5.249373e+005,
+37-12403,CC_ptca1,2.167433e+000,
+37-12403,CC_ptca2,4.847529e-002,
+37-12403,CC_ptcb0,2.540063e+001,
+37-12403,CC_ptcb1,-2.750000e-004,
+37-12403,CC_ptcb2,0.000000e+000,
+37-12403,CC_ptempa0,-6.839104e+001,
+37-12403,CC_ptempa1,5.249233e-002,
+37-12403,CC_ptempa2,-5.873641e-007,
+37-12403,CC_p_range,1450,
```
Comment (reedan88) [inline calibration/CTDMOG/CGINS-CTDMOG-12403__20240124.csv:25]: Looks good.

File: calibration/CTDMOG/CGINS-CTDMOG-12404__20240202.csv [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+37-12404,CC_ptca0,5.253356e+005,
+37-12404,CC_ptca1,4.691704e+000,
+37-12404,CC_ptca2,-6.097520e-002,
+37-12404,CC_ptcb0,2.543287e+001,
+37-12404,CC_ptcb1,-1.025000e-003,
+37-12404,CC_ptcb2,0.000000e+000,
+37-12404,CC_ptempa0,-7.151651e+001,
+37-12404,CC_ptempa1,5.125329e-002,
+37-12404,CC_ptempa2,-5.314104e-007,
+37-12404,CC_p_range,1450,
```
Comment (reedan88) [inline calibration/CTDMOG/CGINS-CTDMOG-12404__20240202.csv:25]: Looks good.

File: calibration/CTDMOG/CGINS-CTDMOG-12405__20240128.csv [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+37-12405,CC_ptca0,5.249723e+005,
+37-12405,CC_ptca1,3.821473e+000,
+37-12405,CC_ptca2,-3.008042e-002,
+37-12405,CC_ptcb0,2.533462e+001,
+37-12405,CC_ptcb1,-2.750000e-004,
+37-12405,CC_ptcb2,0.000000e+000,
+37-12405,CC_ptempa0,-6.782872e+001,
+37-12405,CC_ptempa1,5.025920e-002,
+37-12405,CC_ptempa2,-2.199966e-007,
+37-12405,CC_p_range,1450,
```
Comment (reedan88) [inline calibration/CTDMOG/CGINS-CTDMOG-12405__20240128.csv:25]: Looks good.

File: calibration/CTDMOH/CGINS-CTDMOH-10264__20240128.csv [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+37-10264,CC_ptca0,5.255193e+005,
+37-10264,CC_ptca1,4.454563e+000,
+37-10264,CC_ptca2,-8.227381e-002,
+37-10264,CC_ptcb0,2.521925e+001,
+37-10264,CC_ptcb1,5.000000e-005,
+37-10264,CC_ptcb2,0.000000e+000,
+37-10264,CC_ptempa0,-7.188110e+001,
+37-10264,CC_ptempa1,5.050947e-002,
+37-10264,CC_ptempa2,-3.157414e-007,
+37-10264,CC_p_range,5076,
```
Comment (reedan88) [inline calibration/CTDMOH/CGINS-CTDMOH-10264__20240128.csv:25]: Looks good.

File: calibration/CTDMOH/CGINS-CTDMOH-11704__20240126.csv [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+37-11704,CC_ptca0,5.255849e+005,
+37-11704,CC_ptca1,6.809290e+000,
+37-11704,CC_ptca2,-1.029222e-001,
+37-11704,CC_ptcb0,2.533375e+001,
+37-11704,CC_ptcb1,-2.250000e-003,
+37-11704,CC_ptcb2,0.000000e+000,
+37-11704,CC_ptempa0,-7.242269e+001,
+37-11704,CC_ptempa1,5.059452e-002,
+37-11704,CC_ptempa2,-5.156798e-007,
+37-11704,CC_p_range,5076,
```
Comment (reedan88) [inline calibration/CTDMOH/CGINS-CTDMOH-11704__20240126.csv:25]: Looks good.

File: calibration/CTDMOH/CGINS-CTDMOH-12225__20240128.csv [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+37-12225,CC_ptca0,5.243437e+005,
+37-12225,CC_ptca1,-1.710647e+000,
+37-12225,CC_ptca2,-2.124094e-002,
+37-12225,CC_ptcb0,2.524687e+001,
+37-12225,CC_ptcb1,-1.825000e-003,
+37-12225,CC_ptcb2,0.000000e+000,
+37-12225,CC_ptempa0,-6.753744e+001,
+37-12225,CC_ptempa1,5.077814e-002,
+37-12225,CC_ptempa2,-2.186034e-007,
+37-12225,CC_p_range,5076,
```
Comment (reedan88) [inline calibration/CTDMOH/CGINS-CTDMOH-12225__20240128.csv:25]: Looks good.

File: calibration/CTDMOH/CGINS-CTDMOH-12386__20240131.csv [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+37-12386,CC_ptca0,5.244244e+005,
+37-12386,CC_ptca1,2.909873e+000,
+37-12386,CC_ptca2,-4.018927e-002,
+37-12386,CC_ptcb0,2.500188e+001,
+37-12386,CC_ptcb1,-4.250000e-004,
+37-12386,CC_ptcb2,0.000000e+000,
+37-12386,CC_ptempa0,-7.238393e+001,
+37-12386,CC_ptempa1,5.096546e-002,
+37-12386,CC_ptempa2,-7.137057e-007,
+37-12386,CC_p_range,5076,
```
Comment (reedan88) [inline calibration/CTDMOH/CGINS-CTDMOH-12386__20240131.csv:25]: Looks good.

File: calibration/CTDMOG/CGINS-CTDMOG-12372__20240128.csv
Comment (rtravis-whoi) [reply calibration/CTDMOG/CGINS-CTDMOG-12372__20240128.csv:1]: corrected!

Comment (sauuyer) [reply calibration/CTDMOG/CGINS-CTDMOG-12372__20240128.csv:1]: Change looks good

**Review by sauuyer — APPROVED** [review]
looks good, thanks Rebecca!

**Pull request merged**